src/layout/_helper_classes.scss

```diff
@@ -1,81 +1,38 @@
-// Layout
-//
-// 1. Spree-auth-devise hax which I hate
-//
-// @@@ TODO
-// This file should just be positioning/layout helper classes
+// Layout helper classes
 
-// class for defining sections of content
-.r-section {
-  position: relative;
+.is-invisible {
+  @include is-hidden;
 }
 
-
-// wrapper that contrains regions to max site width
-.r-wrapper {
-  width: $site-max-width;
-}
-
-
-// class for centering sections
-.r-centered {
-  text-align: center;
-}
-
-
-#search-bar {
-  display: none; // [1]
+.is-hidden {
+  display: none;
 }
 
+.is-hidden-on-small-screens {
 
-.r-wrapper-centered {
-  width: $site-max-width;
-  margin-left: auto;
-  margin-right: auto;
-}
-
-.r-inner {
-  max-width: 1080px;
-  margin-left: auto;
-  margin-right: auto;
+  @include respond-below($break-small) {
+    display: none !important;
+  }
 }
 
+.is-hidden-on-large-screens {
 
-.r-section-wrapper {
-  padding: ms(5) ms(2);
+  @include respond-at-and-above($break-small) {
+    display: none !important;
+  }
 }
 
-
-.r-section-wrapper--sm {
-  padding: 40px 23px 30px;
-
-  /*@include respond-between($break-large, $site-max-container) {
-    padding: 30px 50px;
-  }*/
+.clearfix {
+  @include clearfix;
 }
 
-
 .mx-auto {
   margin-left: auto;
   margin-right: auto;
 }
 
-
 .-hoisted {
   @include respond-at-and-above(1200px) {
     margin-top: -40px;
   }
 }
```

Comment:
Let's go ahead and remove lines 29-38. It is old code and we don't have these concepts in core yet. may as well keep this lean.